support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -204,7 +204,7 @@ function userInTest(
 	}
 
 	if (referrerControlled) {
-		return acquisitionDataTest && acquisitionDataTest.name === testId;
+		return !!acquisitionDataTest && acquisitionDataTest.name === testId;
```

Comment:
Did you want this in here?

Reply:
Oops no, I think it will disappear when you merge your PR though!